### DISCUSSION TITLE
Docker image for 1.33

### DISCUSSION THREAD

**Discussion by robvdl** [Koenkk/zigbee2mqtt#18689] [General]
About a month ago Home Assistant started warning about the name of ALL my Zigbee devices.

I see it's solved in Z2M 1.33 and I've been patiently checking every week or so to see when that gets released as a Docker image.

Any heads up how close the 1.33 Docker image is?

We've got till February next year to get the 1.33 Docker image out apparently: "This stops working in version 2024.2.0. Please address before upgrading."

**Comment by Koenkk** [comment]
New release = every 1st of the month

**Comment by robvdl** [comment]
Thank you, I didn't know that, I'll never forget that now :)
